src/loaders/compute_tools/gtdb_tk/gtdb_tk.py

```diff
@@ -122,14 +131,37 @@ def _run_gtdb_tk(
             f"Missing IDs {error_tool_safe_data_ids} that are not found in either "
             f"{GTDB_FILTER_FILE_PATTERN} or {GTDB_FAIL_GENOME_FILE}"
         )
-    
+
     for miss_tool_safe_data_id in miss_tool_safe_data_ids:
         error_message = filtered_or_failed_genome_id_mapping[miss_tool_safe_data_id]
         fatal_tuple = create_fatal_tuple(miss_tool_safe_data_id, ids_to_files, error_message)
         fatal_tuples.append(fatal_tuple)
 
     write_fatal_tuples_to_dict(fatal_tuples, output_dir)
 
+    _process_gtdb_result(output_dir, ids_to_files, summary_files)
+
+
+def _process_gtdb_result(
```

Comment:
this code looks really similar to the checkm2 code. Can it be shared?

Reply:
👍

Reply:
:+1: